tracking-library-integration.md

```diff
@@ -0,0 +1,270 @@
+```javascript
+  // OLD TRACKING
+  var productParams = {
+    id: 312,
+    price: 134.50,
+    name: 'Cool sunglasses',
+    url: 'http://fanstore-johnybravo.com/product/coolsunglasses',
+    image_url: 'http://fanstore-johnybravo.com/product/coolsunglasses/coolsunglasses.jpg',
+    categories: [
+      {
+        id: 9912,
+        name: 'Accessories'
+      },
+      {
+        id: 8009,
+        name: 'Summer'
+      }
+    ]
+  }
+
+  metrilo.event('view_product', productParams);
+
+  // NEW TRACKING
+  var productId = 312; // (required) The ID of the product in the shop's database.
+  metrilo.viewProduct(productId);
+```
+
+#### updateCart ####
+The current state of visitor's cart.
+
+:warning: The `add_to_cart` and `remove_from_cart` events are replaced by `updateCart`. The new `updateCart` call accepts only the current cart products and their quantity as parameters. Metrilo generates the add and remove from cart activities internally.
+```javascript
+  // OLD TRACKING
+  var productParams = {
+    id: 312,
+    price: 134.50,
+    name: 'Cool sunglasses',
+    url: 'http://fanstore-johnybravo.com/product/coolsunglasses',
+    image_url: 'http://fanstore-johnybravo.com/product/coolsunglasses/coolsunglasses.jpg'
+  };
+
+  metrilo.event('add_to_cart', productParams); // add to cart
+
+  var productParams = {
+    id: 111,
+    price: 35.04,
+    name: 'Cool shirt',
+    url: 'http://fanstore-johnybravo.com/product/coolshirt',
+    image_url: 'http://fanstore-johnybravo.com/product/coolshirts/coolshirt.jpg'
+  }
+
+  metrilo.event('add_to_cart', productParams); // add to cart
+  metrilo.event('remove_from_cart', { id: 312 }); // remove from cart
+
+  // NEW TRACKING
+  var products = [ // (required)
+    {
+      productId: 312, // (required) The ID of the product in the shop's database.
+      quantity: 2 // (required) How many pieces of the product the visitor has in the cart.
+    },
+    {
+      productId: 111,
+      quantity: 1
+    }
+  ]
+
+  metrilo.updateCart(products); // add to cart
+
+  var products = [
+    {
+      productId: 312,
+      quantity: 2
+    }
+  ]
+
+  metrilo.updateCart(products) // remove from cart
+```
+:bulb: The `view_cart` has been removed. If you still want to track it, you can use [customEvent](#customEvent).
+
+#### applyTags ####
+Adds tags to a visitor.
+
+```javascript
+  // OLD TRACKING
+  metrilo.event('apply_tags', { tags: ['regular', 'big_spender'] });
+
+  // NEW TRACKING
+  var tags = ['regular', 'big_spender'] // (required) Tag labels that are assigned to users in the Metrilo application.
+  metrilo.applyTags(tags);
+```
+
+#### customEvent ####
+Sends a custom event (anything you want to track).
+
+```javascript
+  // OLD TRACKING
+  metrilo.event('paypal_payment');
+
+  // NEW TRACKING
+  var customEventName = 'paypal_payment' // (required) A custom string that the shop sends to Metrilo.
```

Comment:
There's nothing custom about the string, the event is custom

Reply:
Fix here as well: https://editor.swagger.io/?_ga=2.251836425.1499152574.1535452046-1099712288.1523875565